src/github-release.ts

```diff
@@ -52,7 +52,9 @@ export const defaultChangelogTitles = {
   minor: '🚀  Enhancement',
   patch: '🐛  Bug Fix',
   internal: '🏠  Internal',
-  documentation: '📝  Documentation'
+  documentation: '📝  Documentation',
+  'push-to-master':
+    'The following was pushed directly to master and treated as `patch`'
```

Comment:
I think having consistency between the other titles would be good. Keeping it short and having an emoji to both balance it out and draw attention to it. I think we can also do without calling out that it's a patch on every changelog entry. Just having that in the README would be sufficient (IMO). I'm definitely flexible on that view though. Being explicit is nice sometimes. 

```suggestion
    '⚠️  Pushed to master'
```